### DISCUSSION TITLE
Horizontal Line

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1738] [Q&A] [answered]
Shouldn't Horizontal Line block formatting be
`---`

When I type the three - I only get three of them, not a horizontal line.
Maybe the markdown for the horizontal line is something else?

**Comment by zadam** [comment, marked as answer]
Hi, this needs new CKEditor version which will be released in 0.47.

**Reply by hokutonanto** [reply]
Thanks for the info!
